python/lsst/afw/display/interface.py

```diff
@@ -542,7 +542,26 @@ def __addMissingMaskPlanes(self, mask):
                 self.setDefaultMaskPlaneColor(name, next(colorGenerator))
 
     def mtv(self, data, title="", wcs=None):
-        """Display an `~lsst.afw.image.Image` or `~lsst.afw.image.Mask` on a display
+        """Display an image on a display, with semi-transparent masks
+        overlaid, if available.
+
+        Parameters
+        ----------
+        data : `lsst.afw.image.Exposure` or `lsst.afw.image.MaskedImage` or `lsst.afw.image.Image`
+            Image to display.
+        title : str, optional
+            Title for the display window.
+        wcs : `lsst.afw.geom.SkyWcs`, optional
+            World Coordinate System to align an `~lsst.afw.image.MaskedImage`
+            or `~lsst.afw.image.Image` to; raise an exception if ``data``
+            is an `~lsst.afw.image.Exposure`.
+
+        Raises
+        ------
+        RuntimeError
+            Raised if an Exposure is passed with a non-None wcs.
```

Comment:
Please re-word. This phrasing sounds to me like the exception is raised if you supply an Exposure that has a non-None wcs.